src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -327,6 +328,7 @@
     </Compile>
     <Compile Include="ResourceHandling\*.cs" />
     <Compile Include="GetCompatiblePlatform.cs" />
+    <Compile Include="GetComAssembliesMetadata.cs" />
```

Comment:
The new task should ship only in the Framework version of `Microsoft.Build.Tasks.Core.dll`. It should be excluded from .NET either here in the project file or with an #ifdef in the source file.